container/test/ticket.json

```diff
@@ -0,0 +1,8 @@
+{
+    "uuid": "ticket-id",
+    "size": 6442450944,
+    "url": "file:///images/disk.img",
+    "timeout": 3000,
+    "sparse": true,
+    "ops": ["read", "write"]
+}
```

Comment:
Actually, since we already have tickets in the examples folder, they do not differ, and there is no other process (test or tool) that uses it, I think I will just remove it.